src/client/MwWindow.ts

```diff
@@ -11,8 +11,12 @@ interface MediaWiki {
 	config: MwConfig;
 }
 
+export interface WikibaseContentLanguages {
+	getAllPairs: () => any;
+}
+
 interface Wikibase {
-	getLanguageNameByCode( languageCode: string ): string;
+	WikibaseContentLanguages: any;
```

Comment:
I just learned that you can define construct signatures to hint against classes in TS! https://www.typescriptlang.org/docs/handbook/interfaces.html#difference-between-the-static-and-instance-sides-of-classes

This could then become something like
```suggestion
	WikibaseContentLanguages: {
		new(): WikibaseContentLanguages;
	};
```